bindings/c/CMakeLists.txt

```
@@ -38,8 +38,37 @@ else()
   add_library(fdb_c SHARED ${FDB_C_SRCS} ${asm_file})
   strip_debug_symbols(fdb_c)
 endif()
+
+# cbatrace section 
+if(NOT WIN32 AND NOT APPLE)
```

why not run this on macOS? I understand we currently don't run bindingtester on macOS, but if we could generate it there, it might still be useful.

---

I personally do sometimes run the binding tester on macOS for one-off tests.

---

The bindingtester would still work without this change - it just would report API coverage. So if it is hard to make this work on macOS, I think it would be ok not to run this on macOS. But to me it looks like the above line could just be changed to this and everything would work: 

```suggestion
if(NOT WIN32)
```